backend/wmg/data/transform.py

```diff
@@ -129,6 +130,26 @@ def build_gene_id_label_mapping(gene_ontology_term_ids: List[str]) -> List[dict]
     def build_ontology_term_id_label_mapping(ontology_term_ids: Iterable[str]) -> List[dict]:
         return [{ontology_term_id: ontology_term_label(ontology_term_id)} for ontology_term_id in ontology_term_ids]
 
+    def order_tissues(ontology_term_ids: Iterable[str]) -> Iterable[str]:
+        """
+        Order tissues based on appearance in TissueMapper.HIGH_LEVEL_TISSUES. This will maintain the priority set in
+        that class which is intended to keep most relevant tissues on top and tissues that are related to be placed
+        sequentially
+        """
+        ontology_term_ids = set(ontology_term_ids)
+        ordered_ontology_term_ids = []
+        for tissue in TissueMapper.HIGH_LEVEL_TISSUES:
+            tissue = TissueMapper.make_id_writable(tissue)
+            if tissue in ontology_term_ids:
+                ontology_term_ids.remove(tissue)
+                ordered_ontology_term_ids.append(tissue)
+
+        if ontology_term_ids:
+            ordered_ontology_term_ids = ordered_ontology_term_ids + list(ontology_term_ids)
```

Comment:
I am not sure I fully understand the functionality, but `ontology_term_ids` won't be ordered anymore after you convert them to list.

Reply:
Whatever is left would be orphan tissues, for those the order doesn't really matter